corehq/apps/fixtures/models.py

```diff
@@ -522,3 +523,22 @@ def for_all_item_ids(cls, item_ids, domain):
         ).all()
 
         return ownerships
+
+
+class UserFixtureType(object):
+    LOCATION = "LOCATION"
+    CHOICES = (
+        (LOCATION, "Location")
+    )
+
+
+class UserFixtureStatus(models.Model):
+    """
+    Keeps track of when a user needs to re-sync a fixture
+    """
+    user_id = models.CharField(max_length=100, db_index=True)
+    fixture_type = models.CharField(max_length=50)
```

Comment:
You could also add a choices option to this field https://docs.djangoproject.com/en/1.6/ref/models/fields/#choices although reading the docs, it doesn't look like it actually does any validation around that, unless this ends up with a django admin view or similar.  Yeah, probably fine without.